src/Microsoft.SymbolStore/SymbolStores/HttpSymbolStore.cs

```diff
@@ -69,26 +70,70 @@ public HttpSymbolStore(ITracer tracer, SymbolStore backingStore, Uri symbolServe
+            _client = new HttpClient
+            {
                 Timeout = TimeSpan.FromMinutes(4)
             };
 
-            // If PAT, create authenticated client
-            if (!string.IsNullOrEmpty(personalAccessToken))
+            if (hasPAT)
             {
-                var handler = new HttpClientHandler() {
+                var handler = new HttpClientHandler()
+                {
                     AllowAutoRedirect = false
                 };
-                var client = new HttpClient(handler) {
+                var client = new HttpClient(handler)
+                {
                     Timeout = TimeSpan.FromMinutes(4)
                 };
                 client.DefaultRequestHeaders.Accept.Add(new MediaTypeWithQualityHeaderValue("application/json"));
-                client.DefaultRequestHeaders.Authorization = new AuthenticationHeaderValue(
-                    "Basic", Convert.ToBase64String(ASCIIEncoding.ASCII.GetBytes(string.Format("{0}:{1}", "", personalAccessToken))));
+                // Authorization is set in associated constructors.
                 _authenticatedClient = client;
             }
         }
 
+        /// <summary>
+        /// Create an instance of a http symbol store
+        /// </summary>
+        /// <param name="tracer">logger</param>
+        /// <param name="backingStore">next symbol store or null</param>
+        /// <param name="symbolServerUri">symbol server url</param>
+        /// <param name="personalAccessToken">optional Basic Auth PAT or null if no authentication</param>
+        public HttpSymbolStore(ITracer tracer, SymbolStore backingStore, Uri symbolServerUri, string personalAccessToken = null)
+            : this(tracer, backingStore, symbolServerUri, !string.IsNullOrEmpty(personalAccessToken))
+        {
+            // If PAT, create authenticated client with Basic Auth
+            if (!string.IsNullOrEmpty(personalAccessToken))
+            {
+                _authenticatedClient.DefaultRequestHeaders.Authorization = new AuthenticationHeaderValue("Basic", Convert.ToBase64String(ASCIIEncoding.ASCII.GetBytes(string.Format("{0}:{1}", "", personalAccessToken))));
+            }
+        }
+
+        /// <summary>
+        /// Create an instance of a http symbol store with an authenticated client
+        /// </summary>
+        /// <param name="tracer">logger</param>
+        /// <param name="backingStore">next symbol store or null</param>
+        /// <param name="symbolServerUri">symbol server url</param>
+        /// <param name="authenticationHeader">The header information to use for the AuthenticationHeaderValue</param>
+        public HttpSymbolStore(ITracer tracer, SymbolStore backingStore, Uri symbolServerUri, string scheme, string parameter)
+            : this(tracer, backingStore, symbolServerUri, true)
+        {
+            if (string.IsNullOrEmpty(scheme))
+            {
+                throw new ArgumentException(nameof(scheme));
```

Comment:
>ArgumentException [](http://example.com/codeflow?start=26&length=17)

The single arg overload of `ArgumentException` accepts a message argument rather than an argument name. You want either `ArgumentOutOfRangeException` or `ArgumentNullException`